Group typing.md

```diff
@@ -0,0 +1,39 @@
+# Summary
+>Researchers asked participants to work on a variety of group tasks including group typing to measure collective intelligence. The task had groups typing a complicated document simultaneously with other members inputs shown at a slight delay. 
+
+# References
+> Woolley, A., Chabris, C., Pentland, A., Hashmi, N., & Malone, T. (2010). Evidence for a Collective Intelligence Factor in the Performance of Human Groups. Science, 330(6004).
+>
+> Link: https://www-jstor-org.proxy.library.upenn.edu/stable/40931726?pq-origsite=summon&seq=1#metadata_info_tab_contents
+>
+> Supps: https://science.sciencemag.org/content/sci/suppl/2010/09/29/science.1193147.DC1/Woolley_SOM_Revision_1.pdf
+# Stimuli
+## The visual components
+>None given
+
+## Materials for alternative versions of the experiment 
+>The paper describes a variety of collective intelligence tests, which have nothing to do with group typing, can include if desired.
+
+# Procedure
+>When participants arrived at the laboratory they were randomly assigned to participate in a team or, in the event of overscheduling, randomly selected for exclusion. Before meeting their teams, individuals completed a brief intelligence test (described below). Following the intelligence test, teams began working together on the group tasks in a private laboratory room.
+>
+>Individual intelligence test:
+>
+>Raven’s Advanced Progressive Matrices (RAPM) is a standardized test of general fluid reasoning capacity. Participants were given 10 minutes to complete half of the 36-item Raven’s II test (all of the odd-numbered items). Each item presents a 3 x 3 array (or “matrix”) of shapes with the lower-right corner empty. Based on patterns in the sequences of shapes, subjects must pick which of 8 other shapes properly belongs in the empty space. There is only one correct answer per question. Questions become progressively more difficult as the test goes on. Examples of items are available from Harcourt Education.
+>
+>Group typing:
+>
+>Groups were provided a hard copy of a complicated text and worked for 10 minutes to simultaneously type as much of the text as possible into a shared online document. Participants were each seated in front of a separate computer and worked in the shared online document where they could see each others' work with a slight delay. Teams earned one point for each word correctly typed, and lost one point for each word skipped and for each typo. Team members thus needed to carefully coordinate their work to avoid typing over the work of other members or skipping whole sections (which would result in the loss of many points)
+
+## Roles 
+>Each group member had the same role.
+
+## Instructions
+>None given, searching the literature returns no results either. May reach out to author.
```

Comment:
Did you reach out to the author yet? Since there are so many different issues on this same paper maybe we can ask for all the task instructions in one email.

Reply:
True! Which other issues? I can reach out to the researchers soon

Reply:
Actually, a lot of the issues have merged already. The only one left is Checkers. It's not possible to edit after it's merged, right?

Reply:
Not sure, I'll reach out to Mark

Reply:
Sure. Feel free to hold off (perhaps make a new issue without the template capturing the things you want to fix, so we don't forget about them) and then I can show how to do this in our next call.